modules/wizard/wizard.php

```diff
@@ -0,0 +1,762 @@
+		// If there is no language or only one the media step is displayable.
+		if ( ! $languages || count( $languages ) < 2 ) {
+			return true;
+		}
+		foreach ( $languages as $language ) {
+			$media[ $language->slug ] = $this->model->count_posts(
+				$language,
+				array(
+					'post_type'   => array( 'attachment' ),
+					'post_status' => 'inherit',
+				)
+			);
+		}
+		return count( array_filter( $media ) ) === 0;
+	}
+
+	/**
+	 * Return if the licenses step is displayable
+	 *
+	 * @return bool
+	 * @since 2.7
+	 */
+	public function is_licenses_step_displayable() {
+		$licenses = apply_filters( 'pll_settings_licenses', array() );
+		return count( $licenses ) > 0;
+	}
+	/**
+	 * Setup the wizard page
+	 *
+	 * @since 2.7
+	 */
+	public function setup_wizard_page() {
+		// if Polylang isn't activated wizard plugin can't work.
+		if ( ! defined( 'POLYLANG_VERSION' ) ) {
+			return;
+		}
+
+		if ( ! PLL_Admin_Notices::is_dismissed( 'wizard' ) ) {
+			PLL_Admin_Notices::add_notice( 'wizard', $this->wizard_notice() );
+		}
+		$this->redirect_to_wizard();
+		if ( empty( $_GET['page'] ) || 'mlang_wizard' !== $_GET['page'] ) { // phpcs:ignore WordPress.Security.NonceVerification
+			return;
+		}
+
+		$this->steps = apply_filters( 'pll_wizard_steps', $this->steps );
+		$step  = isset( $_GET['step'] ) ? sanitize_key( $_GET['step'] ) : false; // phpcs:ignore WordPress.Security.NonceVerification
+
+		$this->step = $step && array_key_exists( $step, $this->steps ) ? $step : current( array_keys( $this->steps ) );
+
+		// Call the handler of the step for going to the next step.
+		// Be careful check_admin_referer must be done in each handler.
+		// phpcs:disable WordPress.Security.NonceVerification.Missing
+		if ( ! empty( $_POST['save_step'] ) && isset( $this->steps[ $this->step ]['handler'] ) ) {
+			call_user_func( $this->steps[ $this->step ]['handler'], $this );
+		}
+		// phpcs:enable WordPress.Security.NonceVerification.Missing
+		$this->display_wizard_page();
+		// Ensure nothing is done after including the page.
+		exit;
+	}
+
+	/**
+	 * Adds some admin screens where to display the wizard notice
+	 *
+	 * @param bool   $can_display_notice Whether the notice can be displayed.
+	 * @param string $notice             The notice name.
+	 * @return bool
+	 * @since 2.7
+	 */
+	public function can_display_notice( $can_display_notice, $notice ) {
+		if ( ! $can_display_notice && 'wizard' === $notice ) {
+			$screen = get_current_screen();
+			$can_display_notice = in_array(
+				$screen->base,
+				array(
+					'edit',
+					'upload',
+					'options-general',
+				)
+			);
+		}
+		return $can_display_notice;
+	}
+
+	/**
+	 * Return html code of the wizard notice
+	 *
+	 * @since 2.7
+	 */
+	public function wizard_notice() {
+		ob_start();
+		// PLL_MODULES_INC is correctly defined relatively to the plugin folder.
+		include PLL_MODULES_INC . '/wizard/html-wizard-notice.php'; // phpcs:ignore WordPressVIPMinimum.Files.IncludingFile.UsingCustomConstant
```

Comment:
Hum! I think it was because of the use of PLL_MODULES_INC in an other plugin. Indeed now WPCS sniff doesn't say anything however WPCS Vip Minimum expects absolute path.
So I removed phpcs:ignore comment but I keep the use of absolute path with PLL_MODULES_INC constant.